airbyte-integrations/connectors/source-stock-ticker-api-cdk/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+data:
+  allowedHosts:
+    hosts:
+      - TODO # Please change to the hostname of the source.
```

Comment:
do not forget add allowed host for connector

Reply:
Fixed